marimbabot_description/urdf/marimba.urdf.xacro

```diff
@@ -101,7 +101,7 @@ Changelog:
 
 
 <!-- The bar -->
-<xacro:macro name="bar" params="note parent id pos length" >
```

Comment:
The low frames are active by default, if that's what you mean

Reply:
The default is used for the bottom row (no low frames there), otherwise is set to 1 [here](https://github.com/UHHRobotics22-23/marimbabot/pull/281/files#diff-20fb49e2c5b33e5c29b5f7cb39e71b2085e6906d119e054b22e71df8c53c70ceR184)